models/maespa/R/write.config.MAESPA.R

```diff
@@ -24,8 +24,11 @@
 ##' @author Tony Gardella
 ##-------------------------------------------------------------------------------------------------#
 write.config.MAESPA <- function(defaults, trait.values, settings, run.id) {
-  
-  library(Maeswrap)
+
+  if(!require("Maeswrap")){
+    logger.severe("The Maeswrap package is not installed. 
+                  Please consult PEcAn documentation for install notes")
```

Comment:
would be better if it included the link to the relevant page in the documentation